apps/playwright/tests/applicant/experience.spec.ts

```diff
@@ -20,6 +22,115 @@ test.describe("Experiences", () => {
+    await loginBySub(experiencePage.page, "applicant@test.com");
+
+    await experiencePage.addGovCasualWorkExperience({
+      role,
+      startDate: "2001-01",
+      endDate: "2023-01",
+    });
+
+    await expect(experiencePage.page.getByRole("alert")).toContainText(
+      /successfully added experience/i,
+    );
+  });
+
+  test("Can create goc term or indeterminate work experience", async ({
+    appPage,
+  }) => {
+    const role = `Test add goc term or indeterminate work experience (${uniqueTestId})`;
+    const experiencePage = new ExperiencePage(appPage.page);
+    await loginBySub(experiencePage.page, "applicant@test.com");
+
+    await experiencePage.addGovTermOrIndeterminateWorkExperience({
+      role,
+      startDate: "2001-01",
+      endDate: "2023-01",
+    });
+
+    await expect(experiencePage.page.getByRole("alert")).toContainText(
+      /successfully added experience/i,
+    );
+  });
+
+  test("Can create goc contractor work experience", async ({ appPage }) => {
+    const role = `Test add goc contractor work experience (${uniqueTestId})`;
+    const experiencePage = new ExperiencePage(appPage.page);
+    await loginBySub(experiencePage.page, "applicant@test.com");
+
+    await experiencePage.addGovContractorWorkExperience({
+      role,
+      startDate: "2001-01",
+      endDate: "2023-01",
+    });
+
+    await expect(experiencePage.page.getByRole("alert")).toContainText(
+      /successfully added experience/i,
+    );
+  });
+
+  test("Can create caf work experience", async ({ appPage }) => {
+    const role = `Test add caf work experience (${uniqueTestId})`;
+    const experiencePage = new ExperiencePage(appPage.page);
+    await loginBySub(experiencePage.page, "applicant@test.com");
+
+    await experiencePage.addCafWorkExperience({
+      role,
+      startDate: "2001-01",
+    });
+
+    await expect(experiencePage.page.getByRole("alert")).toContainText(
+      /successfully added experience/i,
+    );
+  });
+
+  // test("Can edit work experience", async ({ appPage }) => {
+  //   const role = `Test edit work experience (${uniqueTestId})`;
+  //   const experiencePage = new ExperiencePage(appPage.page);
+  //   await loginBySub(experiencePage.page, "applicant@test.com");
+
+  //   await experiencePage.addCafWorkExperience({
+  //     role,
+  //     startDate: "2001-01",
+  //   });
+
+  //   await expect(experiencePage.page.getByRole("alert")).toContainText(
+  //     /successfully added experience/i,
+  //   );
+
+  //   const applicantCtx = await graphql.newContext("applicant@test.com");
+  //   const applicant = await me(applicantCtx, {});
+  //   console.log(applicant.workExperiences);
+  //   const workExperience = applicant.workExperiences?.find(
+  //     (ex) => ex?.role === role,
+  //   );
+
+  //   await experiencePage.editWorkExperience(`${workExperience?.id}`, {
+  //     role,
+  //     startDate: "2001-01",
+  //     endDate: "2200-01"
+  //   });
+  // });
```

Comment:
Was this intentional?

Reply:
Nay, thanks for catching that :+1: Fixed in [b117b27](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12208/commits/b117b276c4e300e2bdc782c743fd406e0c086af4)